### ISSUE TITLE
Add other translate engine like PONS

### ISSUE THREAD

**Issue by braisarias** [crow-translate/crow-translate#92]
Do you plan add more translate engines to the project?

I've found some translation mistakes with the engines supported by crow.
Particularly I prefer use PONS translate for some pair of languages. You can find some information about that API here: https://es.pons.com/p/diccionario-en-linea/developers/api

**Comment by Shatur95** [comment]
Yes, I will add other engines, but PONS require registration. Maybe I will add the ability to specify API key for some engines in future.

**Comment by Shatur95** [comment]
Should be done on the QOnlineTranslator side.
